src/aunit/TestRunner.h

```diff
@@ -40,8 +40,8 @@ namespace aunit {
  */
 class TestRunner {
   public:
-    /** Integer type of the timeout parameter. Seconds. */
-    typedef uint8_t TimeoutType;
+    /** Integer type of the timeout parameter. Seconds. Default is kTimeoutDefault = 10 */
+    typedef unsigned int TimeoutType;
```

Comment:
Can you change this to `uint16_t` instead so that the size is deterministic? An `unsigned int` can be 2-bytes (8-bit AVR), 4-bytes (ESP8266, ESP32), or 8-bytes (Linux, MacOS). Yes, I use AUnit on Linux and MacOS for CI/CD.

Secondly, update the PR description with `(fixes #57)` so that the issue is auto-closed when this is submitted.

Reply:
Done ... eventually got it right.